### ISSUE TITLE
(Question) Is it possible to run a FIX server with Quickfix?

### ISSUE THREAD

**Issue by vinifr** [quickfix/quickfix#586]
Hi, I would like to run a FIX server instead a FIX client in order to provide quotations to certain application.

**Comment by Tipmethewink** [comment]
Certainly. I use it for client integration testing using python so haven't stress tested it. Also `examples/executor` and `examples/ordermatch` both use `SocketAcceptor` so are presumably examples of servers.

**Comment by TOliverKocsis** [comment]
Could you please include some sort of step by step guide? Or is there some link / article we could refer to ?

**Comment by Malacarne** [comment]
@TOliverKocsis a very simple example below:

``` c++
int main(int argc, char *argv[])
  {
    Application application;
    FIX::SessionSettings settings("[config_file_here]");
    FIX::FileStoreFactory storeFactory(settings);
    FIX::FileLogFactory logFactory(settings);
    FIX::SocketAcceptor acceptor(application, storeFactory, settings, logFactory);
    
    //
    acceptor.start();
    application.run();
    acceptor.stop();

    //
    return 0;
  }
```

**Comment by orenmnero** [comment]
Yes — `SocketAcceptor` is the server-side (acceptor) component and is covered in the QuickFIX documentation. This is a usage question rather than a bug report; please use GitHub Discussions or the mailing list for support questions. Closing.
